ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthGetWork.java

```diff
@@ -54,7 +54,8 @@ public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
       final String[] result = {
         "0x" + BaseEncoding.base16().lowerCase().encode(rawResult.getPrePowHash()),
         "0x" + BaseEncoding.base16().lowerCase().encode(dagSeed),
-        rawResult.getTarget().toHexString()
+        rawResult.getTarget().toHexString(),
+        "0x" + Long.toHexString(rawResult.getBlockNumber())
```

Comment:
Would it make sense to use `Quantity` here?

Reply:
Nice catch

Reply:
Fixed